dbms/src/Server/Server.cpp

```diff
@@ -269,6 +270,8 @@ std::string Server::getDefaultCorePath() const
     return getCanonicalPath(config().getString("path")) + "cores";
 }
 
+bool PDClientHelper::enable_safepoint_v2 = false;
```

Comment:
Could you please put it into PDClientHelper.cpp? It's a little stange to initialize it here